code/modules/mob/living/carbon/carbon.dm

```diff
@@ -106,6 +106,8 @@
 			Weaken(3)
 	if(override)
 		return override
+	else if(shock_damage > 400)
+		gib()

```

Comment:
electricity explodes you https://youtu.be/coW1RHUsf_I?t=134
the gibs are also a big warning to not touch whatever is next to them